pravega/pravega-client-rust
# Issue 354: Nodejs StreamManager

Signed-off-by: thekingofcity <3353040+thekingofcity@users.noreply.github.com>

**Change log description**  
Add `StreamManager` support to Nodejs bindings.

**Purpose of the change**  
Fixes #354 

**What the code does**  
Use `neon-bindings` to wrap the rust client.

**How to verify it**  
1. `npm i`
2. `npm run build-debug`
3. `node --loader ts-node/esm tests/stream_manager.ts` should pass in a clean Pravega standalone deployment.
